packages/react-core/src/components/Menu/MenuItemAction.tsx

```diff
@@ -45,24 +45,29 @@ const MenuItemActionBase: React.FunctionComponent<MenuItemActionProps> = ({
             onActionClick && onActionClick(event, itemId, actionId);
           };
           return (
-            <button
+            <div
               className={css(
                 styles.menuItemAction,
                 isFavorited !== null && 'pf-m-favorite',
                 isFavorited && styles.modifiers.favorited,
                 className
               )}
-              aria-label={ariaLabel}
-              onClick={onClickButton}
-              {...((isDisabled === true || isDisabledContext === true) && { disabled: true })}
-              ref={innerRef}
-              tabIndex={-1}
               {...props}
             >
-              <span className={css(styles.menuItemActionIcon)}>
-                {icon === 'favorites' || isFavorited !== null ? <StarIcon aria-hidden /> : icon}
-              </span>
-            </button>
+              <Button
+                aria-label={ariaLabel}
+                onClick={onClickButton}
+                ref={innerRef}
+                role="menuitem"
```

Comment:
This should help resolve an aXe error that is related to https://github.com/patternfly/patternfly-react/issues/9968. This also matches the Core markup.

That said, it'd be worth looking into an alternative to making each button inside a menu a `role="menuitem"`. My concern would be whether it's be totally clear that a user can navigate up/down *and* left/right inside a menu, or whether the assumption would be that there's only items in a vertical scope.